config/samples/databricks_v1alpha1_run_job.yaml

```diff
@@ -1,10 +1,9 @@
 apiVersion: databricks.microsoft.com/v1alpha1
```

Comment:
Could you please add second file databricks_v1alpha1_run_job_path.yaml instead of changing databricks_v1alpha1_run_job.yaml? It's good to have two examples.

Reply:
done